benchopt/config.py

```diff
@@ -25,15 +23,36 @@
 }
 
 
+def get_global_config_file():
+    "Return the global config file."
+
+    config_file = os.environ.get('BENCHO_CONFIG', None)
+    if config_file is not None:
+        config_file = Path(config_file)
+        assert config_file.exists(), (
+            f"BENCHO_CONFIG is set but file {config_file} does not exists."
```

Comment:
I would rename to BENCHOPT_CONFIG

Reply:
ok I changed all environment variables to `BENCHOPT_*`.